client/wfnews-war/src/main/angular/src/app/app.component.ts

```diff
@@ -161,7 +161,9 @@ export class AppComponent extends MarkerLayerBaseComponent implements OnDestroy,
                 this.onResize()
             });
         }
-
+            console.log("yes?")
+            // this.store.dispatch(getIncidents('test'))
```

Comment:
And commented out code as well. No sense in cluttering up with stuff we dont need